### ISSUE TITLE
Cache job listings to preserve scroll position

### ISSUE THREAD

**Issue by SkaterDad** [AviKKi/unnwhiteboard#2]
Really cool project 👍 

Enhancement request:  When viewing a job, and then going back to the list page, the data from "load more" isn't cached, so you lose your place.

**Comment by AviKKi** [comment]
Yeah that seems like a very needed UX improvement, on checking it out I saw that all the component state is reset when we come back.

Filter, Location and Jobs List are all reset, this needs to be implemented using a [writable store](https://svelte.dev/tutorial/writable-stores).
